src/components/NewSpaceCwForm.js

```diff
@@ -0,0 +1,152 @@
+  };
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      await axios.post('http://localhost:3001/api/v1/users/:user_id/space_cws', formData);
+      setSuccessMessage('New coworking space created successfully!');
+      setTimeout(() => {
+        setSuccessMessage('');
+      }, 5000); // 5 seconds
+    } catch (error) {
+      setErrorMessage('Error creating coworking space. Please try again later.');
+      setTimeout(() => {
+        setErrorMessage('');
+      }, 8000); // 8 seconds
+    }
+  };
+
+  return (
+    <div className={styles.newCwsContainer}>
+      <div className={styles.newCwsOverlay}>
+        <h2 className={styles.newCwsTitle}>Create a New Coworking Space</h2>
+        <form onSubmit={handleSubmit} className={styles.formContainer}>
+          {/* <label htmlFor="name">Name:</label> */}
+          <input
+            type="text"
+            name="name"
+            className="text"
+            placeholder="Name:"
+            value={formData.name}
+            onChange={handleChange}
+            required
+          />
+
+          {/* <label htmlFor="model">Model:</label> */}
+          <input
+            type="text"
+            name="model"
+            className="text"
+            placeholder="Model:"
+            value={formData.model}
+            onChange={handleChange}
+          />
+
+          {/* <label htmlFor="description">Description:</label> */}
+          <textarea
+            name="description"
+            className="textarea"
+            rows="5"
+            placeholder="Description:"
+            value={formData.description}
+            onChange={handleChange}
+            required
+          />
+
+          {/* <label htmlFor="address">Address:</label> */}
+          <input
+            type="text"
+            name="address"
+            className="text"
+            placeholder="Address:"
+            value={formData.address}
+            onChange={handleChange}
+          />
+
+          {/* <label htmlFor="price">Price:</label> */}
+          <input
+            type="number"
+            name="price"
+            className="number"
+            placeholder="Price: 10.0"
+            value={formData.price}
+            onChange={handleChange}
+          />
+
+          {/* <label htmlFor="image">Image URL:</label> */}
+          <input
+            type="text"
+            name="image"
+            className="text"
+            placeholder="Image URL:"
+            value={formData.image}
+            onChange={handleChange}
+            required
+          />
+
+          {/* <label htmlFor="discount">Discount:</label> */}
+          <input
+            type="number"
+            name="discount"
+            className="number"
+            placeholder="Discount: 10.0"
+            value={formData.discount}
+            onChange={handleChange}
+          />
+
+          {/* <label htmlFor="category">Category:</label> */}
```

Comment:
Hi @kazim110 ! Yo have done a great job so far. Could you please delete these 8 <label> tags that are not used? Thanks.